### PR TITLE
 DTD-2415: Feat: Added tests data to validate against min debt amount for SIA regimeType

### PR DESCRIPTION
**Pull request by thadeusssewanyana**

### CODE REVIEW

File: conf/resources/data/etmp.eligibility.SIA/eligibilityPass.json [hunk 1]
```diff
@@ -61,10 +61,10 @@
 
   "charges": [
     {
-      "chargeType": "VAT Mig Return Debit Charge",
-      "mainType": "VAT Mig Return Charge",
-      "mainTrans": "7700",
-      "subTrans": "1174",
+      "chargeType": "In Year RTI Charge - Tax",
+      "mainType": "In Year RTI Charge (FPS)",
+      "mainTrans": "4920",
```
Comment (rogerball) [inline conf/resources/data/etmp.eligibility.SIA/eligibilityPass.json:66]: Should we use the SIA 4530, 1000 charge?

File: conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceFalse.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      "dueDate": "2022-05-22"
+    }
+  ],
+  "charges": [
+    {
+      "chargeType": "In Year RTI Charge - Tax",
+      "mainType": "In Year RTI Charge (FPS)",
+      "mainTrans": "4920",
+      "subTrans": "1553",
+      "taxPeriodFrom": "2022-02-06",
```
Comment (rogerball) [inline conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceFalse.json:83]: This is SA, should we use the SIA 4530, 1000 charge?

File: conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceTrue.json [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      "toDate": "2022-05-05",
+      "dueDate": "2022-05-22"
+    }
+  ],
+  "charges": [
+    {
+      "chargeType": "In Year RTI Charge - Tax",
+      "mainType": "In Year RTI Charge (FPS)",
+      "mainTrans": "4920",
+      "subTrans": "1553",
```
Comment (rogerball) [inline conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceTrue.json:82]: This is SA, should we use the SIA 4530, 1000 charge?

File: conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceFalse.json [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+      "taxPeriodTo": "2022-03-05",
+      "chargeReference": "XM123456789812",
+      "outstandingAmount": 31.99,
```
Comment (rogerball) [inline conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceFalse.json:86]: In line with the way we have worded other stubs, this should be renamed to isLessThanMinDebtAllowanceTrue, I think

File: conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceTrue.json [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+      "taxPeriodFrom": "2022-02-06",
+      "taxPeriodTo": "2022-03-05",
+      "chargeReference": "XM123456789812",
+      "outstandingAmount": 32.00,
```
Comment (rogerball) [inline conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceTrue.json:86]: In line with the way we have worded other stubs, this should be renamed to isLessThanMinDebtAllowanceFalse, for this value

File: conf/resources/data/etmp.eligibility.SIA/eligibilityPass.json (hunk 1 above)
Comment (thadeusssewanyana) [reply conf/resources/data/etmp.eligibility.SIA/eligibilityPass.json:66]: Spoke to Robyn and @SpencerCGriffiths yesterday. they're "pointing to SA charges for now until SIA implementation ticket".

File: conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceFalse.json (hunk 4 above)
Comment (thadeusssewanyana) [reply conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceFalse.json]: renamed

File: conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceFalse.json (hunk 2 above)
Comment (thadeusssewanyana) [reply conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceFalse.json:83]: Spoke to Robyn and @SpencerCGriffiths yesterday. they're "pointing to SA charges for now until SIA implementation ticket".

File: conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceTrue.json (hunk 3 above)
Comment (thadeusssewanyana) [reply conf/resources/data/etmp.eligibility.SIA/isLessThanMinDebtAllowanceTrue.json:82]: Spoke to Robyn and @SpencerCGriffiths yesterday. they're "pointing to SA charges for now until SIA implementation ticket".

**Review by rogerball — APPROVED** [review]

**Pull request merged**